ibm/data_source_ibm_cbr_rule.go

```diff
@@ -0,0 +1,309 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/platform-services-go-sdk/contextbasedrestrictionsv1"
+)
+
+func dataSourceIBMCbrRule() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMCbrRuleRead,
+
+		Schema: map[string]*schema.Schema{
+			"rule_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The ID of a rule.",
+			},
+			"id": &schema.Schema{
```

Comment:
id is not required.. please remove this block

Reply:
removed block:
```
			"id": &schema.Schema{
				Type:        schema.TypeString,
				Computed:    true,
				Description: "The globally unique ID of the rule.",
			},
```